Assignment/Test.c

```diff
@@ -7,8 +7,26 @@
 // Program on main branch.
 // ------------------------------------------------------------------------------------------------
 #include "Program.h"
+#include <stdio.h>
+
+void DisplayRes (ComplexNumber num) {
+   printf ("%.2f + %.2fi\n", num.real, num.img);
+}
 
 int main () {

```

Comment:
This is a manual test program. You should have written an automated test program. Also the input prompts are not uniformly formatted. You were asked to just print whether the result was correct or not. Instead you have just printed the  complex number without any message, or without proper formatting. You are printing a negative imaginary part with a + sign in front (e.g. 10 + -3i).